mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,7 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
```

Comment:
This works. As a little tip, it's often more readable not to use `else` in a simple 'yes/no' situation like this. So in this case you could write:

```
function mood(isHappy) {

  if (isHappy) {
    return "I am happy";
  } 
  
  return "I am not happy";
}
```

That would work exactly the same because `isHappy` is either true or false. If it's true, then we return out of the function early, so `return "I am not happy"` never executes